chart/templates/iq-server-jobs.yaml

```diff
@@ -37,6 +52,10 @@ spec:
               name: {{ .Release.Name }}-iq-server-secrets-volume
             {{- end }}
           env:
+            {{- range .Values.iq_server.env }}
+            - name: {{ .name }}
+              value: {{ .value | quote }}
+            {{ - end }}
```

Comment:
Ideally we should update the `iq-server-deployment_test.yaml` and `iq-server-jobs_test.yaml` to check that the new values can be set properly

Reply:
Tests are updated.